google/cloud/bigtable/internal/bulk_mutator_test.cc

```diff
@@ -88,7 +91,8 @@ TEST(BulkMutatorTest, Simple) {
                                          std::move(mut));
 
   EXPECT_TRUE(mutator.HasPendingMutations());
-  auto status = mutator.MakeOneRequest(*mock);
+  auto limiter = std::make_shared<bigtable_internal::NoopMutateRowsLimiter>();
```

Comment:
Q.  Should we be using a `MockMutateRowsLimiter` (non-strict, no expectations) in these tests instead?

Reply:
I considered that. I know that I don't like the warnings if we used a normal mock. `NiceMock<MockMutateRowsLimiter>` vs. `NoopMutateRowsLimiter` seems mostly the same to me.

I might give the slightest edge to using `NoopMutateRowsLimiter` because that is what our actual code will be doing most of the time. (In this PR, we use it all of the time).